### PR TITLE
Update to golang 1.19

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
The only specific reason to update is to have [0] in that automatically increases the number of open file descriptors we can have. I didn't updated to 1.20 because the base docker images we use don't have this version available yet.

### CODE REVIEW

File: pkg/client/clientset/versioned/fake/register.go [hunk 1]
```diff
@@ -35,14 +35,14 @@ var localSchemeBuilder = runtime.SchemeBuilder{
...
+//	import (
+//	  "k8s.io/client-go/kubernetes"
+//	  clientsetscheme "k8s.io/client-go/kubernetes/scheme"
+//	  aggregatorclientsetscheme "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/scheme"
+//	)
 //
-//   kclientset, _ := kubernetes.NewForConfig(c)
-//   _ = aggregatorclientsetscheme.AddToScheme(clientsetscheme.Scheme)
+//	kclientset, _ := kubernetes.NewForConfig(c)
+//	_ = aggregatorclientsetscheme.AddToScheme(clientsetscheme.Scheme)
```
Comment (alban) [inline pkg/client/clientset/versioned/fake/register.go:45]: Should that be in a separate PR? Ditto for register.go

**Review by alban — APPROVED** [review]
LGTM, modulo the change to generated files.

Thanks for fixing the issue

File: pkg/client/clientset/versioned/fake/register.go (hunk 1 above)
Comment (mauriciovasquezbernal) [reply pkg/client/clientset/versioned/fake/register.go:45]: I can move that to a different commit, however this should be on the same PR otherwise the tests won't pass.

Comment (alban) [reply pkg/client/clientset/versioned/fake/register.go:45]: Oh ok, I didn't realise.
